### PR TITLE
Added BFS & DFS

### PR DESCRIPTION
**Pull request by akashksinghal**

### CODE REVIEW

**Comment by akashksinghal** [comment]
Added Queues and Stacks

File: Algorithms/Circular_Queue/CircularQueue.cpp [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+		for (int i = 0; i <= rear; i++) 
+			cout<<arr[i]<<" "; 
+	} 
+} 
+
+int main() 
+{ 
+	Queue q(5); 
+    int N=1;
+    do
```
Comment (Bhavnaharitsa) [inline Algorithms/Circular_Queue/CircularQueue.cpp:97]: mention what N stands for..avoid giving single letter variables and use sensible names

File: Algorithms/Circular_Queue/CircularQueue.cpp [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            cout<<"\nDeleted value = "<<q.deQueue(); 
+            cout<<endl;
+            q.displayQueue();
+        }
+    }while(N!=3);
+    q.displayQueue();
+    /*
+	q.enQueue('A'); 
+	q.enQueue('B'); 
+	q.enQueue('C');
```
Comment (Bhavnaharitsa) [inline Algorithms/Circular_Queue/CircularQueue.cpp]: delete commented code

File: Algorithms/Stacks/Stack in Array.cpp [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                cout<<A[top--]<<endl;
+            }
+            else
+                cout<<"Stack EMPTY";
+        }
+        else
+            if(top!=-1)
+				cout<<A[top]<<endl;
+			else
+				cout<<"EMPTY "<<endl;
```
Comment (Bhavnaharitsa) [inline Algorithms/Stacks/Stack in Array.cpp]: use brackets for if and else following the syntax

**Review by akashksinghal — COMMENTED** [review]
Updated As said

**Review by akashksinghal — COMMENTED** [review]
Updated As Said

**Review by akashksinghal — COMMENTED** [review]
Resolved

**Pull request merged**